dll/win32/shell32/CIDLDataObj.cpp

```diff
@@ -198,12 +198,33 @@ HRESULT WINAPI CIDLDataObj::Initialize(HWND hwndOwner, PCIDLIST_ABSOLUTE pMyPidl
 
     m_cfShellIDList = RegisterClipboardFormatW(CFSTR_SHELLIDLIST);
 
-    FORMATETC HIDAFormat = { (CLIPFORMAT)m_cfShellIDList, NULL, DVASPECT_CONTENT, -1, TYMED_HGLOBAL };
+    FORMATETC Format = { (CLIPFORMAT)m_cfShellIDList, NULL, DVASPECT_CONTENT, -1, TYMED_HGLOBAL };
     STGMEDIUM medium = {0};
     medium.tymed = TYMED_HGLOBAL;
     medium.hGlobal = hida;
+    HRESULT hr = SetData(&Format, &medium, TRUE);
+    if (!FAILED_UNEXPECTEDLY(hr) && bAddAdditionalFormats)
```

Comment:
Newer windows have these,
not sure if they are loaded on demand or added automatically.

Currently our shell needs the CF_HDROP (in way too many places, it should use HIDA)